packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -47,8 +46,10 @@ contract Bridge is EssentialContract, IBridge {
      * External Functions*
      *********************/
 
-    /// Allow Bridge to receive ETH from EtherVault.
-    receive() external payable {}
+    /// Allow Bridge to receive ETH from the TokenVault or EtherVault.
+    receive() external payable {
+        // TODO(dave): require the sender is the TokenVault or EtherVault?
```

Comment:
I think we can add this check here.

Reply:
was having trouble with the unit tests, so i abandoned it for the time being. i will leave a todo with a branch.